prime-router/settings/staging/0017-add-md-organization.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: "md-phd"
+  description: "Maryland Public Health Department"
```

Comment:
Same comment.  This is already in staging under 'md-doh', so let's be careful not to duplicate.

I created a 0017-maryland.yml file but you beat me to it, getting the PR in.